sycl/test/basic_tests/parallel_for_range_roundup.cpp

```diff
@@ -0,0 +1,113 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST SYCL_OPT_PFWGS_TRACE=1 %t.out 2> %t.errout
+// RUN: %ACC_RUN_PLACEHOLDER SYCL_OPT_PFWGS_TRACE=1 %t.out 2> %t.errout
+// RUN: %CPU_RUN_PLACEHOLDER SYCL_OPT_PFWGS_TRACE=1 %t.out 2> %t.errout
+// RUN: %GPU_RUN_PLACEHOLDER SYCL_OPT_PFWGS_TRACE=1 %t.out 2> %t.errout
+// RUN: cat %t.errout | FileCheck %s
```

Comment:
I'm not sure if that naked FileCheck will work here.  Usually the run placeholder is followed by check placeholder, so in case a device isn't present on a test machine, that we don't try to run a check that might not have a run.  
For example:
`RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER`
This has tripped me up before. It might be why you have the failure on CUDA. Not sure.

Reply:
ok